jaggery-modules/rxt/module/scripts/permissions/permissions.js

```diff
@@ -946,6 +946,36 @@ var permissions = {};
         }
         return authorized;
     };
+
+    permissions.SYSTEM_ANON_ROLE = 'SYSTEM/wso2.anonymous.role';
+    permissions.checkActionPermissionforPath = function(path,action,tenantId,username) {
```

Comment:
Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/e95caa53ae6d2bec4d278537d014af0496cdfa7a)